src/app/ui/modal-dialog-custom.js

```diff
@@ -11,10 +11,13 @@ module.exports = {
   alert: function (text) {
     modal('', yo`<div>${text}</div>`, null, { label: null })
   },
-  prompt: function (title, text, inputValue, ok, cancel) {
+  prompt: function ({ title, text, inputValue, multiline }, ok, cancel) {
```

Comment:
doing that you break other calls to `prompt` ... could you just make it another function in here please ?